test/integration/config-experimental-warning/test/index.test.js

```diff
@@ -143,6 +143,20 @@ describe('Config Experimental Warning', () => {
     expect(stdout).toMatch(' · cpus: 2')
   })
 
+  it('should show the configured value for string features', async () => {
+    configFile.write(`
+      module.exports = {
+        experimental: {
+          ppr: 'incremental'
+        }
+      }
+    `)
+
+    const stdout = await collectStdoutFromDev(appDir)
+    expect(stdout).toMatch(experimentalHeader)
+    expect(stdout).toMatch(' · ppr: "incremental"')
```

Comment:
nice drive-by addition

Reply:
gonna be hard to find this specific part of CLI change by searching with PR title later tho..

Reply:
Move to #80686.